packages/clarity-visualize/src/interaction.ts

```diff
@@ -34,7 +34,13 @@ export class InteractionHelper {
         let de = doc.documentElement;
         let scrollTarget = this.layout.element(data.target as number) as HTMLElement || doc.body;
         let scrollable = scrollTarget.scrollHeight > scrollTarget.clientHeight || scrollTarget.scrollWidth > scrollTarget.clientWidth;
-        if (scrollTarget && scrollable) { scrollTarget.scrollTo(data.x, data.y); }
+        if (scrollTarget && scrollable) {
+            scrollTarget.scrollTo(data.x, data.y);
```

Comment:
Suggestion: What is this doing? Add comment to explain this as not self-evident.

Reply:
This was a user reported bug. In some cases, because of active CSS, documentElement is not scrollable and therefore we need to fall back to body. I will add comments.